images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -349,6 +349,12 @@ function Build-PackageManagementEnvironmentTable {
             "Value" = $env:VCPKG_INSTALLATION_ROOT
         }
     )
+    $envVariables += @(
+        @{
+            "Name" = "VCPKG_ROOT"
+            "Value" = $env:VCPKG_ROOT
+        }
+    )
```

Comment:
we can add this ennv variable in the section above, no need to update the #envVariables

Reply:
In general, yes, it's redundant.